modules/extensions/src/translations/en.json

```diff
@@ -146,5 +146,11 @@
     "message": "The requested event was not found. Possible reasons:",
     "message2": "The Event Collector is not enabled in Botpress Config",
     "message3": "The event was pruned from the database"
+  },
+  "state": {
+    "variable": "Variable",
+    "valueBefore": "Value Before",
+    "valueAfter": "Value After",
```

Comment:
I think "Current Value" would be clearer here

Reply:
Agreed but `Value After` is what's in the design